build.sbt

```diff
@@ -1,7 +1,7 @@
 name := "Nautilus-Cloud-Metering"
 version := "0.1"
 
-scalaVersion := "2.12.10"
+scalaVersion := "2.12.11"
```

Comment:
What does this bring?
If useful, why not 2.12.12 directly to make it more future-safe?

Reply:
*.12 launched 3 days ago, do we consider it stable? If so, i will bump it to *.12. 

I am genuinely asking as I don't tend to follow their release cycle closely.

Reply:
I didn't realise it was so recent, but they are generally mostly bug-fix or back-ports from 2.13.

it's not a first release, I'm not sure that waiting for bugs to manifest would lead to more stability. They would be fixed in a 2.12.13 which will be as good as this as far as stability is concerned...

Reply:
Sure, bumped.